### PR TITLE
Fixed publisher permissions

### PR DESCRIPTION
**Pull request by dnkemper**
# How to test
Make sure I added back publisher permissions I removed previously: https://github.com/uiowa/uiowa/pull/3663/files

### CODE REVIEW

File: config/ighn.international.uiowa.edu/user.role.publisher.yml [hunk 1]
```diff
@@ -64,12 +64,15 @@ permissions:
   - 'create terms in research_areas'
   - 'create terms in tags'
   - 'create twitter media'
-  - 'create url aliases'
```
Comment (cory-skeers) [inline config/ighn.international.uiowa.edu/user.role.publisher.yml:67]: Was removing this intentional?

Comment (dnkemper) [reply config/ighn.international.uiowa.edu/user.role.publisher.yml:67]: ahhh forgot to push up fix to that!  Haha 100% was not.

File: config/policy.clas.uiowa.edu/user.role.publisher.yml [hunk 2]
```diff
@@ -68,13 +68,16 @@ permissions:
   - 'create terms in research_areas'
   - 'create terms in tags'
   - 'create twitter media'
-  - 'create url aliases'
```
Comment (cory-skeers) [inline config/policy.clas.uiowa.edu/user.role.publisher.yml:71]: And this one ;)

**Review by cory-skeers — APPROVED** [review]
Perfect! 💯 💯

**Comment by dnkemper** [comment]
Thank you so much, @cory-skeers

**Pull request merged**